crt_portal/cts_forms/views.py

```diff
@@ -178,6 +178,74 @@ def post(self, request, id):
         return render(self.request, 'forms/complaint_view/show/index.html', output)
 
 
+class ProFormView(SessionWizardView):
+    @method_decorator(login_required)
+    def dispatch(self, *args, **kwargs):
```

Comment:
Opportunity here to leverage Django's [`LoginRequiredMixin`](https://docs.djangoproject.com/en/3.0/topics/auth/default/#the-loginrequired-mixin) instead of overriding `dispatch`